LearnositySDK/Request/Init.cs

```diff
@@ -191,7 +197,7 @@ private string generateRequestString()
         /// - the `action` value if passed
         /// </summary>
         /// <returns>A signature hash for the request authentication</returns>
-        private string generateSignature()
+        public string generateSignature()
```

Comment:
Turned to `public` so we can use it in the tests. In Java SDK this is also a `public` method.